faq.md

```diff
@@ -0,0 +1,58 @@
+---
+icon: light-bulb
+label: FAQs
+order: 100
+---
+
+# Frequently Asked Questions
+
+==- Who should use Votre?
+Votre, like other borrowing protocols, will predominantly be used by holders of concentrated token positions such as family offices, high net worth individuals, foundations, DAOs, miners, validators, traders, degens and more.
+
+==- How do you prevent liquidations?
+Votre hedges directional risk upfront so as to remove the need for forced sales in illiquid market environments. This is done in a way that does not introduce counterparty risk or credit risk from a borrower.
+
+==- How do you hedge collateral?
+The protocol swaps the collateral to the borrower’s chosen liability token (typically stablecoins), mitigating the protocol’s exposure to any decline in the asset’s price.
+
+==- But what if the asset appreciates in value?
+Providers post stablecoins upfront to guarantee the borrower’s upside exposure to the cap.
+
+==- Why should providers use Votre?
+Because Votres are volatile bets and providers can monetize this volatility. They do this by offering terms to the user that contain a small spread, then monetizing this spread over the life of the trade. This technique is called “gamma scalping” and you can learn more about it here.
+
+==- How do I know I’m getting the best price?
+Votre allows for open competition between providers onchain to provide the best offer. With every Votre request via the hosted frontend, users are shown competitive quotes before they can execute in order to promote best execution.
+
+==- How big is this “spread”?
+Like any spread, it is a function of the number of providers. Votre will go live with three sophisticated options providers from day one who we have partnered with, and they will compete for user flow.
+
+==- Why would I cap my returns?
+Because in bull markets it can be beneficial to take chips off the table or even redeploy borrowed collateral into a new position for greater capital efficiency. Users can also “roll” their positions higher as assets appreciate to maintain exposure so long as there is sufficient provider-provided liquidity.
+
+==- How is the final price determined?
+Uniswap TWAP oracles
+
+==- What venues does the protocol swap on?
+Uniswap for now, but this may expand in the future as liquidity improves
+
+==- Does swapping create slippage?
+Yes, but compared to interest rates and capital requirements of other protocols, this is minimal.
```

Comment:
This depends on trade size vs. the available liquidity. It's possible slippage is negligible for typical trade sizes. There is however swap fee, typically 0.05% for the pools currently configured.